page-container/index-page/ContactForm.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Typography, Grid, makeStyles, TextField, Button
+} from '@material-ui/core';
+import { useCallback, useState } from 'react';
+import packageConfig from '../../package.json';
+
+const useStyle = makeStyles((theme) => ({
+  title: {
+    color: '#ffffff'
+  },
+  formRoot: {
+    marginTop: '7rem',
+    width: '28.75rem',
+    '& > div': {
+      marginBottom: '3rem',
+      '&:last-child': {
+        marginTop: '1.8125rem',
+        marginBottom: 0
+      }
+    },
+    '& input, & textarea, & label': {
```

Comment:
Generally I prefer using style-components overal material ui stylings if the thing we want to do is not too material ui specific, but ok to get it working first

Reply:
I'll take some time to learn the styled-component for styling. I could refactor styling later on.